Update To 1.8.7

Fabricio20 opened the pull request:
Update this plugin to Spigot 1.8.7 ( NMS 1.8.3 ).

LeeMcNeil commented:
Hey @Fabricio20, thank you for the PR! Could you please explain the changes and I'll alert our Java developer to take a look.

Fabricio20 commented:
Changes Made:
- Fixed All Bukkit.getOnlinePlayers Methods ( New Versions Use Collection< extends Player > wich is not supproted by the old Player Array ( Player[] ) methods );
- Removed All The Useless Deprecation Nodes wich are not nescessary anymore;
- Fixed static access on Plugin Variable;
- Removed unused imports;
- Removed/Commented unused variables.
